spring-boot-project/spring-boot-actuator-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -1926,7 +1926,7 @@
     },
     {
       "name": "management.metrics.graphql.autotime.enabled",
-      "description": "Whether to automatically time web client requests.",
+      "description": "Whether to automatically time GraphQL requests.",
```

Comment:
I've opened https://github.com/spring-projects/spring-boot/issues/43901 for this as I think it warrants a separate issue that'll appear in the changelog.